documentation/src/main/asciidoc/entity-view/manual/en_US/04_mappings.adoc

```diff
@@ -766,24 +767,25 @@ public interface CatView {
     @MappingCorrelated(
         correlationBasis = "age",
         correlator = PersonAgeCorrelationProvider.class,
-        correlationResult = "pers"
+        fetch = FetchStrategy.JOIN
     )
     Set<Person> getSameAgedPersons();
 
     static class PersonAgeCorrelationProvider implements CorrelationProvider {
 
         @Override
         public void applyCorrelation(CorrelationBuilder builder, String correlationExpression) {
-            builder.correlate(Person.class, "pers") # <1>
-                .on("pers.age").inExpressions(correlationExpression) # <2>
+            final String alias = builder.getCorrelationAlias();  # <1>
+            builder.correlate(Person.class)
+                .on(alias + ".age").inExpressions(correlationExpression) # <2>
             .end();
         }
 
     }
 }
 ----
-<1> Currently the alias _leaks_ into the main query builder when using the `JOIN` strategy so try to use a unique name
-<2> Represents the `correlationBasis`. We generally recommend to use the `IN` predicate through `inExpressions()` to be able to easily switch the fetch strategy
+<1> The alias returned by `getCorrelationAlias()` is what is going to be used for the correlated entity
```

Comment:
This is better rephrased to:
"`getCorrelationAlias()` defines the alias for the correlated entity"